tutor/templates/local/docker-compose.jobs.yml

```diff
@@ -29,7 +29,7 @@ services:
         {%- for mount in iter_mounts(MOUNTS, "openedx", "lms-job") %}
         - {{ mount }}
         {%- endfor %}
-      depends_on: {{ [("mysql", RUN_MYSQL), ("mongodb", RUN_MONGODB)]|list_if }}
+      depends_on: {{ [("mysql", RUN_MYSQL), ("mongodb", RUN_MONGODB), ("meilisearch", RUN_MEILISEARCH)]|list_if }}
```

Comment:
I find it weird that we never caught this issue when we were running elasticsearch. What's the error stacktrace? And can you please add a changelog entry?

Reply:
ES did not have an explicit init job if I remember correctly.